webxr/xrSession_features_deviceSupport.https.html

```diff
@@ -31,34 +31,27 @@
     }
 
     return navigator.xr.test.simulateDeviceConnection(fakeDeviceInitParams)
-      .then((controller) => new Promise((resolve, reject) => {
-        navigator.xr.test.simulateUserActivation(() => {
+      .then((controller) => navigator.xr.test.simulateUserActivation(() => 
```

Comment:
This last part here shouldn't be needed should it? You can just directly do the new method?

Reply:
cc @Manishearth

Reply:
Probably yes

Reply:
Fixed